plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/steps/BuildLambda.kt

```diff
@@ -4,13 +4,16 @@
 package software.aws.toolkits.jetbrains.services.lambda.steps
 
 import com.intellij.execution.configurations.GeneralCommandLine
+import org.junit.AfterClass
```

Comment:
this code is in the `src` tree, which generally means it is not appropriate to reference any libraries that are used in tests. in an upcoming PR, we pull in some build system improvements that remove a lot of test libraries from the classpath, which will make this harder to do

Reply:
sure, will provide in the test class